gpkit/nomials.py

```diff
@@ -536,42 +537,80 @@ def mono_approximation(self, x0):
                         "it's already a Monomial." % str(self))
 
 
-class Constraint(Posynomial):
-    """A constraint of the general form monomial >= posynomial
-    Stored internally (exps, cs) as a single Posynomial (1 >= self)
-    Usually initialized via operator overloading, e.g. cc = (y**2 >= 1 + x)
-    Additionally retains input format (lhs vs rhs) in self.left and self.right
-    Form is self.left >= self.right.
+from abc import ABCMeta, abstractproperty
+
+class OneEQConstraint(object):
```

Comment:
not sure about the naming here -- what's a `OneEQConstraint`? And does this belong in `nomials`?